setup.py

```diff
@@ -8,32 +8,37 @@
 PACKAGE = "pypcap"
 VERSION = "1.1.5"
 
+
 def recursive_search_dirs(dirs, target_files):
     """Recursive search directories"""
     for d in dirs:
         r = recursive_search(d, target_files)
         if r:
             return r
 
+
 def recursive_search(path, target_files):
     """Recursively search for files"""
     for root, _dirs, files in os.walk(path):
         for filename in files:
             if filename in target_files:
                 return os.path.join(root, filename)
 
+
 def get_extension():
     # A list of all the possible search directories
     dirs = ['/usr', sys.prefix] + glob.glob('/opt/libpcap*') + \
         glob.glob('../libpcap*') + glob.glob('../wpdpack*') + \
-        glob.glob('/Applications/Xcode.app/Contents/Developer/Platforms/MacOSX.platform/Developer/SDKs/*')
+        glob.glob('/Applications/Xcode.app/Contents/Developer/Platforms/' +
+                  'MacOSX.platform/Developer/SDKs/*')
 
     for d in dirs:
-        # This makes sure that we first search inside of */include/pcap
-        search_dirs = [os.path.join(d, 'usr', 'include', 'pcap'),
-                    os.path.join(d, 'include', 'pcap'),
-                    os.path.join(d, 'local', 'include', 'pcap'),
-                    d]
+        search_dirs = [
+            os.path.join(d, 'local', 'include'),
+            os.path.join(d, 'usr', 'include'),
+            os.path.join(d, 'include'),
+            d
+        ]
```

Comment:
This will increase the time it takes to find the pcap headers. 
Are there cases in which it will not be in `*/include/pcap`, but be in `*/include`?